tests/phpunit/tests/theme.php

```diff
@@ -282,23 +283,28 @@ public function test_switch_theme() {
 
 		for ( $i = 0; $i < 3; $i++ ) {
 			foreach ( $themes as $name => $theme ) {
+				// Skip invalid theme directory names (such as `block_theme-[0.4.0]`).
+				if ( ! preg_match( '/^[a-z0-9-]+$/', $theme['Stylesheet'] ) ) {
+					continue;
+				}
```

Comment:
Why is this needed?

Reply:
@spacedmonkey This test is extremely old and honestly, poorly written as it includes lots of assertions for wildly different things (unit test standards back then were not what they are today).

We shouldn't have to completely update the tests to cover all the new requirements, new and more appropriate tests should be added for that.

This change here specifically is because some new test theme directories were added that include disallowed characters which make the tests below fail. That was never tested in this test, so we don't have to start now.